ngi_visualizations/fpkm_scatter/fpkm_scatter.py

```diff
@@ -151,11 +195,9 @@ def plot_fpkm_scatter (sample_1, sample_2, x_lab, y_lab, output_fn=False, linear
     Returns a dict containing filenames of PNG and PDF graphs as a dict for 
     each sample.
     """
-    
     # Output filename
     if output_fn is False:
```

Comment:
Can change to `if output_fn is None:` in combination with removing your new code above..